hypothesis-python/tests/array_api/test_arrays.py

```diff
@@ -296,6 +296,26 @@ def test_may_not_fill_unique_array_with_non_nan(xp, xps):
         check_can_generate_examples(strat)
 
 
+def test_floating_point_array():
+    import warnings
+    from hypothesis.extra.array_api import make_strategies_namespace
+
+    try:
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            import numpy.array_api as nxp
+    except ModuleNotFoundError:
+        import numpy as nxp
+    xps = make_strategies_namespace(nxp)
+    dtypes= xps.floating_dtypes() | xps.complex_dtypes()
```

Comment:
Looks like our Python 3.8 tests don't support the newer version of the array api standard with complex numbers; I'd be inclined to just `@pytest.mark.skipif(sys.version_info[:2] <= (3, 8), reason="no complex")` since we'll be dropping 3.8 when it goes EOL in October anyway.